exodus_lambda/functions/origin_request.py

```diff
@@ -103,10 +102,9 @@ def handler(self, event, context):
                 )
                 content_type = query_result["Items"][0]["content_type"]["S"]
                 if content_type:
-                    query = urllib.parse.urlencode(
-                        {"ResponseContentType": content_type}
-                    )
-                    request["uri"] += "?" + query
+                    request[
+                        "querystring"
+                    ] = "response-content-type={0}".format(content_type)
```

Comment:
As mentioned here: https://github.com/release-engineering/exodus-lambda/pull/105#discussion_r465682860
If we want to protect against special characters aside from "/";
```suggestion
                    request[
                        "querystring"
                    ] = "response-content-type=%s" % urllib.parse.quote(content_type)
```

But if slashes are the only special character ever used here, then I guess there's no reason for encoding.

At the very least, though, I'd suggest we use the older "%" means of interpolation, just to be consistent with what's already used throughout the repository.
```suggestion
                    request[
                        "querystring"
                    ] = "response-content-type=%s" % content_type
```